internal-registry/redhat-developer/web-terminal-dev/4.5.0/devworkspacetemplate.yaml

```diff
@@ -0,0 +1,27 @@
+kind: DevWorkspaceTemplate
+apiVersion: workspace.devfile.io/v1alpha2
+metadata:
+  name: web-terminal-dev
+spec:
+  components:
+    - name: web-terminal
+      container:
+        image: "${RELATED_IMAGE_plugin_redhat_developer_web_terminal_dev_4_5_0}"
+        command: ["/go/bin/che-machine-exec",
+                  "--authenticated-user-id", "$(DEVWORKSPACE_CREATOR)",
+                  "--idle-timeout", "$(DEVWORKSPACE_IDLE_TIMEOUT)",
+                  "--pod-selector", "controller.devfile.io/workspace_id=$(CHE_WORKSPACE_ID)",
+                  "--use-bearer-token"]
+        endpoints:
+          - name: web-terminal
+            targetPort: 4444
+            secure: true
+            exposure: public
+            protocol: http
+            attributes:
+              type: ide
+              cookiesAuthEnabled: "true"
```

Comment:
Is it still relevant? and after we introduce POST method there, it might be not safe.
So, I just propose to remove unused stuff for time being.

Reply:
:+1: